README.md

```diff
@@ -347,6 +347,13 @@ prefix {
   path = "foo/bar"
 }
 
+prefix {
```

Comment:
Why a separate `prefix` section instead of including this in the one above?

Reply:
Fair point. I wrote this a long time ago so I can't remember my original rationale so I'll put it above